lxd/instance_exec.go

```diff
@@ -542,6 +542,11 @@ func instanceExecPost(d *Daemon, r *http.Request) response.Response {
 		return response.BadRequest(err)
 	}
 
+	// Contraint validation.
+	if post.RecordOutput && post.WaitForWS {
```

Comment:
Should we also reject post.Interactive with post.RecordOutput?

Reply:
Yes, we should.